.github/workflows/build-test-release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - id: matrix
-        uses: splunk/addonfactory-test-matrix-action@v1
```

Comment:
Let's revert this change as well. Let's stick to the latest possible version of Splunk.

Reply:
Yes, will do that.

Reply:
Done